src/app/@ansyn/ansyn/modules/plugins/app/alerts/alerts.plugin.ts

```diff
@@ -83,9 +87,10 @@ export class AlertsPlugin extends BaseImageryPlugin {
 			const viewExtent = position.extentPolygon;
 			const intersection = getPolygonIntersectionRatio(viewExtent, overlay.footprint);
 			isInBound = Boolean(intersection);
-			action = !isInBound ? of(new AddAlertMsg(payload)) :
-				this.outOfBound ? of(new RemoveAlertMsg(payload)) : of(null);
+			action = !isInBound ? of(new BackToExtentAction()) :
```

Comment:
There is a problem with doing the position check here
This plugin is for alerts. We cancelled the alert. So the check on the extent does not seem in place here any more. 
We can move the position check to another service / effect.
But, if you think that we will need this alert again, I suggest to leave the alert in the code (including the files that you deleted) and just add the new action (back to extent) without removing the old action (add alert msg)